src/core/app_config/postinit.ts

```diff
@@ -0,0 +1,15 @@
+import { i18n } from '~core/localization';
+
+export function getDefaultFeedObject(feed?: string) {
```

Comment:
this file  is never imported
forgotten to remove ?

Reply:
hmm, look like we lost some functionality after rebase

Reply:
ok, so need to be rechecked after returning

Reply:
Fixed. Not the best solution, but i add task about think where to write such logic
https://kontur.fibery.io/search/ZKZ06#Tasks/Task/Fix-cycled-deps-between-localization-service-and-config-13554